package/CHANGELOG

```diff
@@ -53,6 +56,7 @@ Changes
   * The "AMBER" entry in setup.py's extra_requires has now been removed. Please
     use `pip install ./package[extra_formats]` instead (PR #3810)
   * `Universe.empty` emmits less warnings (PR #3814)
+  * adding element attribute to TXYZParser if all atom names are valid element symbols (PR #3826)
```

Comment:
Interesting, I'm not sure why this showing up in the diff. Could you try to update your branch against develop - it's hard to see what's part of this now merged PR and what is the code being added.

Reply:
I'm old at rebasing and I always mess this up too 😅 Let us know if you need any input!